streampark-console/streampark-console-service/src/main/resources/flink-application.conf

```diff
@@ -15,60 +15,61 @@
 # limitations under the License.
 #
 
-flink:
-  deployment:
-    property: #@see: https://ci.apache.org/projects/flink/flink-docs-release-1.12/deployment/config.html
-      $internal.application.main:
-      pipeline.name:
-      yarn.application.queue:
-      taskmanager.numberOfTaskSlots: 1
-      parallelism.default: 2
-      jobmanager.memory:
-        flink.size:
-        heap.size:
-        jvm-metaspace.size:
-        jvm-overhead.max:
-        off-heap.size:
-        process.size:
-      taskmanager.memory:
-        flink.size:
-        framework.heap.size:
-        framework.off-heap.size:
-        managed.size:
-        process.size:
-        task.heap.size:
-        task.off-heap.size:
-        jvm-metaspace.size:
-        jvm-overhead.max:
-        jvm-overhead.min:
-        managed.fraction: 0.4
-      pipeline:
-        auto-watermark-interval: 200ms
-      # checkpoint
-      execution:
-        checkpointing:
-          mode: EXACTLY_ONCE
-          interval: 30s
-          timeout: 10min
-          unaligned: false
-          externalized-checkpoint-retention: RETAIN_ON_CANCELLATION
-      # state backend
-      state:
-        backend: hashmap # Special note: flink1.12 optional configuration ('jobmanager', 'filesystem', 'rocksdb'), flink1.12+ optional configuration ('hashmap', 'rocksdb'),
-        backend.incremental: true
-        checkpoint-storage: filesystem
-        savepoints.dir: file:///tmp/chkdir
-        checkpoints.dir: file:///tmp/chkdir
-      # restart strategy
-      restart-strategy: fixed-delay  # Restart strategy [(fixed-delay|failure-rate|none) a total of 3 configurable strategies]
-      restart-strategy.fixed-delay:
-        attempts: 3
-        delay: 5000
-      restart-strategy.failure-rate:
-        max-failures-per-interval:
-        failure-rate-interval:
-        delay:
-  # table
-  table:
-    planner: blink # (blink|old|any)
-    mode: streaming #(batch|streaming)
```

Comment:
Hi @wolfboys , thanks for your contribution.

Why delete table related configs?

Reply:
> Hi @wolfboys , thanks for your contribution.
> 
> Why delete table related configs?

oh, These are not standard parameters in Flink, These parameters can be set in `env.property `